justfile

```diff
@@ -314,8 +312,18 @@ ln-dlc-node-test args="": docker
 
 # Runs background Docker services
 docker:
+    #!/usr/bin/env bash
+    set -euxo pipefail
     docker-compose up -d
 
+    # wait a few seconds to ensure that Docker containers started
+    sleep 2
+
+    docker exec -it bitcoin bitcoin-cli --regtest -rpcuser=admin1 -rpcpassword=123 createwallet "test"
+    address=$(docker exec -it bitcoin bitcoin-cli --regtest -rpcuser=admin1 -rpcpassword=123 getnewaddress)
+    address="${address%%[[:cntrl:]]}"
+    docker exec bitcoin bitcoin-cli --regtest -rpcuser=admin1 -rpcpassword=123 generatetoaddress 300 $address
```

Comment:
NIT: I wouldn't generate 300 blocks but only the necessary amount (101). Generating 300 blocks will take (a bit) longer for no value.